fdk-mqa/fdk-mqa-vocabulary.ttl

```diff
@@ -0,0 +1,286 @@
+@prefix dqv:    <http://www.w3.org/ns/dqv#> .
+@prefix xsd:    <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos:   <http://www.w3.org/2004/02/skos/core#> .
+@prefix dcat:   <http://www.w3.org/ns/dcat#> .
+@prefix foaf:   <http://xmlns.com/foaf/0.1/> .
+@prefix dct:     <http://purl.org/dc/terms/> .
+@prefix cc:    <http://creativecommons.org/ns#> .
+@prefix digdir:     <https://data.norege.no/vocabulary/fdk-mqa#> .
```

Comment:
Litt usikker her på to ting:
1. Bør vi bruke `digdir` som prefix,
2. `fdk` i namespace.

Kva med:
1. `dcatno-mqa`
2. `https://data.norge.no/vocabulary/dcatno-mqa`

PS! Det er ein typo i det som står: "norege".